msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMsalController.java

```diff
@@ -667,18 +656,19 @@ public void run() {
      * this needs to be called on background thread.
      */
     @Override
-    public List<AccountRecord> getAccounts(@NonNull final OperationParameters parameters)
+    public List<ICacheRecord> getAccounts(@NonNull final OperationParameters parameters)
             throws ClientException, InterruptedException, ExecutionException, RemoteException, OperationCanceledException, IOException, AuthenticatorException {
         final String methodName = ":getBrokerAccounts";
         if (isMicrosoftAuthServiceSupported(parameters.getAppContext())) {
             Logger.verbose(TAG + methodName, "Is microsoft auth service supported? " + "[yes]");
             Logger.verbose(TAG + methodName, "Get the broker accounts from auth service.");
-            return getBrokerAccountsWithAuthService(parameters);
+            //return getBrokerAccountsWithAuthService(parameters);
         } else {
             Logger.verbose(TAG + methodName, "Is microsoft auth service supported? " + "[no]");
             Logger.verbose(TAG + methodName, "Get the broker accounts from Account Manager.");
-            return getBrokerAccountsFromAccountManager(parameters);
+            //return getBrokerAccountsFromAccountManager(parameters);
         }
+        return new ArrayList<>(); // TODO remove after implementation
```

Comment:
TODO light this up once @rpdome's FLW changes are ready

Reply:
These are invoked in MultipleAccountPCA though. SingleAccountPCA only uses getCurrentAccount()

Reply:
This is not for FLW, but we need to change the result bundle type from AccountRecord to CacheRecord from MicrosoftAuthService.getAccounts().

Reply:
Yes, this is a big TODO -- not necessary for Teams build, but definitely necessary